README.md

```diff
@@ -1586,7 +1586,7 @@ runtime {
 }
 ```
 
-Defaults to "us-central1-b"
+Defaults to the configuration setting `genomics.default-zones` in the JES configuration block which in turn defaults to using `us-central1-b`
```

Comment:
Suggest you add this to line 1705 too (i.e. the place that lists all of the JES runtime attributes (seen in https://github.com/broadinstitute/cromwell/pull/1796)